data-viewer/src/app/view/generic/user_number/[userNumber]/[filename]/page.tsx

```diff
@@ -0,0 +1,33 @@
+import NexusViewer from "@/components/NexusViewer";
+import "../../../../../globals.css";
+import TextViewer from "@/components/TextViewer";
+
+export default function GenericUserDataPage({
+                                                params,
+                                            }: {
+    params: { userNumber: string; filename: string };
+}) {
+    // We expect a route of /generic/experiment_number/filename
+    // This will result in a slug list of [experiment_number, filename]
```

Comment:
```suggestion
    // We expect a route of /generic/user_number/filename
    // This will result in a slug list of [user_number, filename]
```
This should fix it :)